lib/core/templates/error-content/error-content.component.stories.ts

```diff
@@ -0,0 +1,145 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { ErrorContentComponent } from './error-content.component';
+import { CoreStoryModule } from '../../testing/core.story.module';
+import { TemplateModule } from '../template.module';
+import { ActivatedRoute } from '@angular/router';
+import { MatButtonModule } from '@angular/material/button';
+import { of } from 'rxjs';
+
+export default {
+    component: ErrorContentComponent,
+    title: 'Core/Template/Error Content',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, TemplateModule, MatButtonModule],
+            providers: [
+                {
+                    provide: ActivatedRoute,
+                    useValue: { params: of({}) }
+                }
+            ]
+        })
+    ],
+    argTypes: {
+        errorCode: {
+            type: 'string',
+            description: 'Component level Error Code',
+            table: {
+                category: 'Component Inputs',
+                type: {
+                    summary: 'string'
+                },
+                defaultValue: {
+                    summary: 'UNKNOWN'
+                }
+            },
+            defaultValue: '404'
+        },
+        errorCodeTranslated: {
+            type: 'string',
+            description:
+                'Code of translated Error - if translation doesn\'t exist then is UNKNOWN',
+            table: {
+                category: 'Component Variables',
+                type: {
+                    summary: 'string'
+                }
+            },
+            control: {
+                disable: true
+            }
+        },
+        isAdditionalContent: {
+            type: 'boolean',
+            description: 'Enable Content Projection',
+            defaultValue: false,
+            table: {
+                category: 'Story Controls',
+                type: {
+                    summary: 'boolean'
+                },
+                defaultValue: {
+                    summary: false
+                }
+            }
+        }
+    }
+} as Meta;
+
+export const errorCodeStory: Story = args => ({
+    props: args,
+    template: `
+    <adf-error-content [errorCode]="${args.errorCode}">
+        <div adf-error-content-actions *ngIf="${args.isAdditionalContent}">
+        <button mat-raised-button type="button">MyAction</button>
+        </div>
+    </adf-error-content>`
+});
+
+const templateArgTypes = {
+    errorCode: {
+        control: {
+            disable: true
+        }
+    }
+};
+
+const template: Story = args => ({
```

Comment:
can we add type for args and Story? 
```suggestion
const template: Story<ErrorContentComponent> = (args: ErrorContentComponent) => ({
```

Reply:
Done